CorsixTH/Lua/dialogs/resizables/hotkey_assign.lua

```diff
@@ -487,21 +487,23 @@ function UIHotkeyAssign:UIHotkeyAssign(ui, mode)
 
   self:showKeyPane("global")
 
-  -- "Accept" button
+  -- "Cancel" button (rename to "Back" and move to left)
   self:addBevelPanel(self.panel_pos_table_x[1], 430, 200, 40, col_bg)
-      :setLabel(_S.hotkey_window.button_accept)
-      :makeButton(0, 0, 180, 40, nil, self.buttonAccept)
-      :setTooltip(_S.tooltip.hotkey_window.button_accept)
-  -- Reset to defaults button.
+      :setLabel(_S.hotkey_window.button_back)
+      :makeButton(0, 0, 180, 40, nil, self.buttonCancel)
```

Comment:
Suggest renaming the function too (otherwise you have `back` in some places and `cancel` in others).

Reply:
`UIHotkeyAssign:buttonCancel()` --> `UIHotkeyAssign:buttonBack()`

`:makeButton(0, 0, 180, 40, nil, self.buttonBack)